### ISSUE TITLE
Voice Cloning Pause 

### ISSUE THREAD

**Issue by communicaters** [JarodMica/ai-voice-cloning#74]
2024-04-02 17:42:00 | INFO | httpx | HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK"
[Training] [2024-04-02T17:44:28.993175] 24-04-02 17:44:28.979 - INFO: Start training from epoch: 0, iter: 0
[Training] [2024-04-02T17:44:33.395450] [2024-04-02 17:44:33,394] torch.distributed.elastic.multiprocessing.redirects: [WARNING] NOTE: Redirects are currently not supported in Windows or MacOs.
[Training] [2024-04-02T17:44:37.376547] [2024-04-02 17:44:37,376] torch.distributed.elastic.multiprocessing.redirects: [WARNING] NOTE: Redirects are currently not supported in Windows or MacOs.
[Training] [2024-04-02T17:44:42.149943] E:\SD\ai-voice-cloning-v2_0\ai-voice-cloning\runtime\lib\site-packages\torch\optim\lr_scheduler.py:136: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
[Training] [2024-04-02T17:44:42.149943]   warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
[Training] [2024-04-02T17:44:49.001106] E:\SD\ai-voice-cloning-v2_0\ai-voice-cloning\runtime\lib\site-packages\torch\utils\checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
[Training] [2024-04-02T17:44:49.001106]   warnings.warn(
[Training] [2024-04-02T17:46:06.262947] Error no kernel image is available for execution on the device at line 167 in file D:\ai\tool\bitsandbytes\csrc\ops.cu
[Training] [2024-04-02T17:46:09.752771]
[Training] [2024-04-02T17:46:09.752771] E:\SD\ai-voice-cloning-v2_0\ai-voice-cloning>pause
